server/selection_test.go

```diff
@@ -325,10 +325,9 @@ func TestMinLSSelector_SelectUnknownSession_UniformWeights(t *testing.T) {
 func TestMinLSSelector_SelectUnknownSession_SameAddressDivideStake(t *testing.T) {
```

Comment:
Could we update the name of this test since we're no longer dividing stake for the same address?